stable/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "newrelic-infra.fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if ne $name .Release.Name -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Generate basic labels */}}
+{{- define "newrelic-infra.labels" }}
+app: {{ template "name" . }}
+heritage: {{.Release.Service }}
+release: {{.Release.Name }}
+{{- if .Values.podLabels}}
```

Comment:
I would keep pod labels separately because standard labels should go on all resources, not only pod.

Reply:
See above. I'm applying the standard labels to all resources, only adding `chart` when it is safe to do so.

Reply:
Sorry to be picky. Labels `chart` and `heritage` should not go on pods.

See common chart for reference of a label template: https://github.com/kubernetes/charts/blob/HEAD/incubator/common/templates/_metadata_labels.tpl#L21-L27

Also see https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md. 

Or look at the initial chart you get with `helm create`.

Reply:
fixed as per your advice.